hardware_interface/src/joint_state_handle.cpp

```diff
@@ -21,18 +21,13 @@
 
 namespace hardware_interface
 {
-
 JointStateHandle::JointStateHandle()
-: name_(),
-  pos_(nullptr),
-  vel_(nullptr),
-  eff_(nullptr)
-{}
```

Comment:
why these changes? They seem unrelated

Reply:
I really don't know, as I put in the opening post, 
> There are some changes in this PR that should be credited to uncrustify...

Sure, I'll revert 'em